OpenRA.Mods.Common/Widgets/Logic/ServerCreationLogic.cs

```diff
@@ -90,6 +90,10 @@ public ServerCreationLogic(Widget widget, ModData modData, Action onExit, Action
 			checkboxUPnP.OnClick = () => allowPortForward ^= true;
 			checkboxUPnP.IsDisabled = () => !Game.Settings.Server.AllowPortForward;
 
+			var labelUPnP = panel.GetOrNull<LabelWidget>("UPNP_NOTICE");
+			if (labelUPnP != null)
+				labelUPnP.IsVisible = () => !Game.Settings.Server.DiscoverNatDevices;
```

Comment:
Elaborating on my last comment after checking our settings code, I don't think this is helpful.  `DiscoverNatDevices` is the manual override that very few people will ever touch.  The confusion from users comes from `AllowPortForward` being automatically disabled by the game.